linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords_etc_shadow/rule.yml

```
@@ -21,11 +21,17 @@ rationale: |-
 
 severity: high
 
+identifiers:
+    cce@sle15: CCE-91155-2
+    cce@sle12: CCE-83249-3
```

Now we have a test fail:

```
72/223 Test  #73: fix_rules-sort_subkeys ...................................................***Failed    9.92 sec
Number of modified rules: 1
/__w/content/content/linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords_etc_shadow/rule.yml has one or more unsorted references
```
That's because the identifiers and references should be sorted.

Just swap these two lines. 
```suggestion
    cce@sle12: CCE-83249-3
    cce@sle15: CCE-91155-2
```